### ISSUE TITLE
Tales from the trenches: being an open source maintainer.

### ISSUE THREAD

**Issue by nojaf** [amplifying-fsharp/sessions#49]
### Session on Tales from the trenches: being an open source maintainer.

Florian, from Nojaf in CEST would like to tackle:
I'd like to talk a bit about the importance of creating community for a project. How to be a good maintainer. What worked in the past for me. And why it is important to make a project larger than the individual.

**Comment by aguluman** [comment]
Has this session happened yet?

**Comment by nojaf** [comment]
No, we keep certain proposals like this one as a backup in case we cannot find anyone to lead a session. This is currently not scheduled either.

**Comment by aguluman** [comment]
Okay.
Thanks for sharing this information.
